batch/batch/front_end/templates/job.html

```diff
@@ -107,11 +109,11 @@ <h2>Step Status</h2>
         {% endif %}
       </td>
       {% if step['short_error'] is not none %}
-      <td class="data-table-bad" >{{ step['short_error'] }}</td>
+      <td class="data-table-bad">{{ step['short_error'] }}</td>
       {% else %}
       <td></td>
       {% endif %}
-      <td {% if step['state'] in ['failed', 'error'] %}class="data-table-bad"{% endif %}>
+      <td {% if step['state'] in ['failed', 'error' ] %}class="data-table-bad" {% endif %}>
```

Comment:
Can you fix the formatting here so it's back to the original?

Reply:
When I try adding it and save it, it removes it.